### DISCUSSION TITLE
Firmware update - XModem block error 0x021

### DISCUSSION THREAD

**Discussion by schmitto29** [kpine/zwave-js-server-docker#32] [General]
Hi,

i tryed to update my Z-Pi 7 Module over ssh(on computer) on HA.
But when i press "Crtl+A s" during the upload "C" prozess, i get the following error message:
Serial upload aborted
XModem block error 0x021

what i am doing wrong?

Any hints?

**Comment by kpine** [comment]
0x21 means:

> The bootloader did not find the expected start of header (SOH) character at the beginning of the XModem frame.

You didn't say how you are trying to update. Are you doing it by yourself? Using the host method, or docker method described in the wiki? Be sure `sx` or `lsx` are installed. Otherwise, I don't know how to solve your issue besides trying again.

If you continue to have trouble, I'd recommend using Z-Wave JS UI instead.

**Reply by schmitto29** [reply]
With Z-Wave JS UI no problem, thank for the hint.
